### PR TITLE
Implemented Card views for recycler view as well as minimum bid being displayed

### PR DESCRIPTION
**Pull request by StephenPacker**
Try testing before you merge maybe, looks good on my end but not sure how it would look on a phone

### CODE REVIEW

**Comment by StephenPacker** [comment]
Note I only implemented this new adapter for search tasks and view my tasks (I think), need to slightly alter for view tasks bidded on

**Comment by cmbuhler** [comment]
Looks Good! Seems to be lagging when scrolling through the views.
I am receiving many "Skipped 35 frames!  The application may be doing too much work on its main thread." type errors in logcat well scrolling. Also idk how I like the R for requested but that may be just me!

File: app/src/main/java/ca/ualbert/cs/tasko/TaskListAdapter.java [hunk 1]
```diff
@@ -62,27 +85,66 @@ public TaskListAdapter(Context context, TaskList dmtasks){
...
+        // Tries to get the minimum bid on each task if it exists
+        try{
+            BidList bids = dm.getTaskBids(currentTask.getId(), thiscontext);
+            Bid lowbid = bids.getMinBid();
+            if(lowbid != null){
+                String lowbidValue = Float.toString(lowbid.getValue());
+                holder.taskLowestBid.setText("Lowest Bid: " + lowbidValue);
+            }else holder.taskLowestBid.setText("Make the First Bid!");
+        } catch (NoInternetException e){
+            Toast t = new Toast(thiscontext);
```
Comment (cmbuhler) [inline app/src/main/java/ca/ualbert/cs/tasko/TaskListAdapter.java]: This is not the proper way to make toasts. I found this out the hard way. Should be something like this:
`Toast.makeText(getApplicationContext(),"Text", Toast.LENGTH_SHORT).show();`

File: app/src/main/res/layout/task_cardview_layout.xml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    android:id="@+id/TaskSearchDetails"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <android.support.v7.widget.CardView
+        android:id="@+id/taskCardView"
+        android:layout_width="match_parent"
+        android:layout_height="140dp"
+        android:layout_marginBottom="3dp"
```
Comment (cmbuhler) [inline app/src/main/res/layout/task_cardview_layout.xml]: Slightly larger margins may be nice. Or I can shrink mine in view details. Just to be consistent looking  they should be the same. Also verify the Bottom and top both being set does not cause a margin double the desired size (It did in mine that is why I have bottom margin to be 0)

**Comment by cmbuhler** [comment]
This Needing to be mreged or is it WIP? IE should we close but not accept?

**Pull request merged**